tools/deploy/ChronoLog/local_single_user_deploy.sh

```diff
@@ -438,26 +434,25 @@ usage() {
     echo "Examples (Assume installing a Debug build to ~/chronolog_install):"
     echo ""
     echo "  1) Builds ChronoLog in Debug mode and installs it to ~/chronolog_install:"
-    echo "     $0 --build --build-type Debug --install-dir ~/chronolog_install"
+    echo "     $0 --build -t Debug -I ~/chronolog_install"
```

Comment:
better to be consistent in the examples to use long args like `--build-type` and `--install-dir`.